apps/webhook/src/.env.production

```diff
@@ -0,0 +1,7 @@
+NODE_ENV=prod
+PORT=3003
+REDIS_PORT=6379
+REDIS_DB_INDEX=2
+WIDGET_BASE_URL=https://widget.novu.co
+
+STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
```

Comment:
@davidsoderberg is there a need to have this key here in this service? I believe we don't use the provider credentials anywhere in this service. So maybe can we remove it from this service?

Reply:
and provide `{}` to all providers as credentials?

Reply:
We can thing about doing the method `static` so it won't require initializing the class itself. Wdyt?

Reply:
and make the id of the providers static as well to be able to map the providers?

Reply:
Sounds good by me